.github/workflows/build.yml

```diff
@@ -0,0 +1,106 @@
+name: Build and deploy to Kuroco front
+on:
+  push:
+    branches:
+      - main
+      - nuxt3
```

Comment:
Don't we need to add  `nuxt3` here?
Because once nuxt3 subbranch gets updated it overwrites prod deployment.

Reply:
@yabe-diverta san 
Just confirming, 
you mean to remove `nuxt3` from here ?

Reply:
Yes